test/es6/toPrimitive.js

```diff
@@ -159,6 +215,283 @@ var tests = [
             assert.areEqual(-1,Number(a),"should now call @@toPrimitive and return a");
         }
     },
+    {
+        name: "ToPrimitive calling user-defined [@@toPrimitive] method",
+        body: function ()
+        {
+            var o = {}, o1 = {}, o2 = {};
+
+            var retVal, hint;
+
+            o[Symbol.toPrimitive] = function(h) { hint.push(h); return retVal; }
+            o1[Symbol.toPrimitive] = function(h) { hint.push("o1:"+h); return retVal; }
+            o2[Symbol.toPrimitive] = function(h) { hint.push("o2:"+h); return retVal; }
+
+            // Ensuring OrdinaryToPrimitive is not called
+            var desc_toString = Object.getOwnPropertyDescriptor(o, "toString");

```

Comment:
No - removed in latest rev.

---
In reply to: [71950652](https://github.com/Microsoft/ChakraCore/pull/1319#discussion_r71950652) [](ancestors = 71950652)